src/GriffendorFaculty.java

```diff
@@ -0,0 +1,35 @@
+public class GriffendorFaculty {
+
+
+
+
+
+    private String Name;
+
+    public GriffendorFaculty(String name) {
+        Name = name;
+    }
+
+
+    public void CompareStudents(GriffendorStudent firstStudent, GriffendorStudent secondStudent){
```

Comment:
Имена методов с маленькой буквы
```suggestion
    public void сompareStudents(GriffendorStudent firstStudent, GriffendorStudent secondStudent){
```